smokey/src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -208,6 +214,10 @@ public EulerAngle getEulerAngle(){
       swerveNavx.getYaw() + this.yawOffsetDegrees);
   }
 
+  public void addVisionMeasurement(Pose2d RobotPos, double Timestamp){
```

Comment:
This needs to be private and moved down.

Reply:
done